gui_qt/check_report.py

```
@@ -240,6 +240,20 @@ def summarize_manifest_writeback(manifest: dict[str, object]) -> WritebackSummar
             manifest_path=manifest_path,
         )
 
+    apply_failure_path = manifest.get("last_apply_failure_report_path")
+    if isinstance(apply_failure_path, str) and apply_failure_path.strip():
+        facts = [format_manifest_path_fact(manifest_path)] if manifest_path else []
+        facts.append(f"写回失败报告：{apply_failure_path.strip()}")
```

Ruff (0.15.18)

[warning] String contains ambiguous `：` (FULLWIDTH COLON). Did you mean `:` (COLON)?

(RUF001)